src/fft.js

```diff
@@ -91,6 +91,17 @@ define(function (require) {
    *  </code></div>
    */
   p5.FFT = function(smoothing, bins) {
+    Object.defineProperty(this, 'bins', {
+      get: function () {
+        return this.analyser.fftSize / 2;
```

Comment:
I'll close one PR and merge the two branches down to one and modify the remaining PR? :D